docs/studios/unity/tutorials/github_actions.md

```diff
@@ -0,0 +1,309 @@
+---
+sidebar_label: 'Testing with GitHub Actions'
+---
+
+# Setting up QA Bots with GitHub Actions
+
+This tutorial will walk you through setting up a QA Bot for your Unity project using GitHub Actions. This tutorial assumes you have
+some familiarity with [GitHub Actions](https://docs.github.com/en/actions), and that you have a working bot within your
+game (see more on this within the [Creating your first bot guide](/studios/unity/unity-sdk/creating-bots/configuration)). 
+You can find a working example in [this repository](google.com) for reference.
+
+:::caution
+
+This guide and reference is subject to change, as the product is in an **early preview**. We
+are making changes based on your recommendations and feedback, so let us know what you think at
+info@regression.gg, or join our [Discord](https://discord.com/invite/925SYVse2H).
+
+:::
+
+:::info
+
+This features requires a license for Unity, since CI/CD builds for Unity are only available if you have
```

Comment:
I'll let @abeizer suggest new wordings for these sentences as she is good at that, but they read a bit awkwardly as currently written.